mne/transforms.py

```diff
@@ -59,8 +59,12 @@ def _to_const(cf):
         if cf not in _str_to_frame:
             raise ValueError('Unknown cf %s' % cf)
         cf = _str_to_frame[cf]
-    elif not isinstance(cf, int):
-        raise TypeError('cf must be str or int, not %s' % type(cf))
+    else:
+        try:
+            cf = int(cf)
```

Comment:
Actually I think we already have a function like this somewhere that converts coordinate frame str or int to int. Can you check around, probably in `mne/transforms.py`?